app.json

```diff
@@ -18,6 +18,14 @@
     "DISQUS_SITE": {
       "required": true,
       "value": "daostack-alchemy-staging"
+    },
+    "BICONOMY_DAPP_ID":{
```

Comment:
Where is this used?  Does it also need to be defined in the webpack files?

Reply:
Yes DAPP_ID needs to be deleted